### PR TITLE
Cross-compile aarch64

### PR DESCRIPTION
**Pull request by xhochy**
The `linux-aarch64` build failed on `main`. I'm moving this over to Azure as I generally trust the builds there much more than the ones on Azure.

### CODE REVIEW

**Comment by xhochy** [comment]
@conda-forge/torchvision This is ready for review/merge.

File: conda-forge.yml [hunk 1]
```diff
@@ -9,6 +9,5 @@ github:
   tooling_branch_name: main
 os_version:
   linux_64: cos7
-provider:
-  linux_aarch64: default
-test_on_native_only: true
+# Emulated tests exceed the 6h CI threshold
+test: native
```
Comment (h-vetinari) [inline conda-forge.yml:13]: I would prefer just skipping the test suite invocation on aarch, so we at least still have the import tests.

**Review by h-vetinari — COMMENTED** [review]
Thanks for fixing this. Just a small change, then it LGTM

File: conda-forge.yml (hunk 1 above)
Comment (xhochy) [reply conda-forge.yml]: Adjusted.

**Review by h-vetinari — APPROVED** [review]
Thanks a lot!

**Pull request merged**